app/cmd/controller.go

```diff
@@ -83,6 +83,15 @@ func ControllerCmd() cli.Command {
 				Value:    5,
 				Usage:    "HTTP client timeout for replica file sync server",
 			},
+			cli.IntFlag{
+				Name:  "snapshot-max-count",
+				Value: 250,
+				Usage: "Maximum number of snapshots to keep",
+			},
+			cli.StringFlag{
+				Name:  "snapshot-max-size",
+				Usage: "Maximum total snapshot size in bytes or human readable 42kb, 42mb, 42gb",
```

Comment:
What values this exactly handle? KB, KiB or case-insensitive?

Reply:
We use docker/go-units to parse the string. All formats are supported.

https://github.com/docker/go-units/blob/47565b4f722fb6ceae66b95f853feed578a4a51c/size.go#L34